tests/optimal_interpolation/test_testdata.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+import pytest
+import xarray
+
+from xhydro.optimal_interpolation.functions.testdata import (  # noqa: F822
+    _default_cache_dir,
+    get_file,
+    open_dataset,
+)
+
+
+class TestRemoteFileAccess:
+    git_url = "https://github.com/Mayetea/xhydro-testdata"
```

Comment:
@Mayetea Do you have push/write access to `hydrologie/xhydro-testdata`?

It would be better if you targeted a branch of that repo, rather than a fork.

Reply:
Not useful anymore now that Pooch is the main method.